static-analysis.datadog.yml

```diff
@@ -0,0 +1,4 @@
+rulesets:
+  - python-code-style
+  - python-best-practices
+  - python-inclusive
```

Comment:
@therve if you're open to it, would be great if you would be willing to try out the `python-security` ruleset as well because our product offers both code quality and security (SAST) rules